### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by louwers** [brian-team/brian2#918]
I'm getting lots of deprecation warnings when using brian2:

```
WARNING    /home/bart/.local/lib/python3.6/site-packages/brian2/core/variables.py:174: FutureWarning: Conversion of the second argument of issubdtype from `bool` to `np.generic` is deprecated. In future, it will be treated as `np.bool_ == np.dtype(bool).type`.
  return np.issubdtype(np.bool, self.dtype)
 [py.warnings]
WARNING    /home/bart/.local/lib/python3.6/site-packages/brian2/core/variables.py:174: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  return np.issubdtype(np.bool, self.dtype)
 [py.warnings]
WARNING    /home/bart/.local/lib/python3.6/site-packages/brian2/monitors/statemonitor.py:287: FutureWarning: Conversion of the second argument of issubdtype from `int` to `np.signedinteger` is deprecated. In future, it will be treated as `np.int64 == np.dtype(int).type`.
  if np.issubdtype(dtype, np.int):
 [py.warnings]
WARNING    /home/bart/.local/lib/python3.6/site-packages/brian2/monitors/statemonitor.py:57: FutureWarning: Conversion of the second argument of issubdtype from `int` to `np.signedinteger` is deprecated. In future, it will be treated as `np.int64 == np.dtype(int).type`.
  if np.issubdtype(dtype, np.int) and not isinstance(item, np.ndarray):
 [py.warnings]
WARNING    /home/bart/.local/lib/python3.6/site-packages/brian2/groups/group.py:258: FutureWarning: Conversion of the second argument of issubdtype from `int` to `np.signedinteger` is deprecated. In future, it will be treated as `np.int64 == np.dtype(int).type`.
  elif not np.issubdtype(index_array.dtype, np.int):
 [py.warnings]
```

**Comment by mstimberg** [comment]
Thanks for the report. You get the warnings because you are using numpy 1.14, which had not been released yet when we released the latest Brian version. We've already fixed this in Brian's development version (see #913), the problem will therefore disappear with the next Brian release.
You therefore have to workarounds at the moment. Either downgrade numpy to 1.13, or use Brian directly from github master (or, ignore the warnings ;) ).
